provider/cluster/inventory.go

```diff
@@ -245,7 +249,7 @@ loop:
 			// inventory check returned
 
 			runch = nil
-			t.Reset(inventoryResourcePollPeriod)
+			t.Reset(is.config.InventoryResourcePollPeriod)
```

Comment:
invalid operation: is.config (variable with invalid type) has no field or method InventoryResourcePollPeriod